plugins/Marketplace/templates/_updateCommunicationEmail.twig

```diff
@@ -10,14 +10,14 @@
 {% if hasThemeUpdate %}
 <p>
 {{ 'CoreUpdater_NotificationClickToUpdateThemes'|translate }}<br/>
-{{ host }}index.php?module=CorePluginsAdmin&action=themes
+<a href="{{ host }}index.php?module=CorePluginsAdmin&action=themes">{{ host }}index.php?module=CorePluginsAdmin&action=themes</a>
```

Comment:
@aloxe we probably need to escape the first `{{ host }}` like this: `{{ host|e('html_attr') }}`. Also in the link below for better security.

Reply:
sorry for the delay, I haven't noticed this comment. I pushed the change just now.